kedro-datasets/kedro_datasets/svmlight/svmlight_dataset.py

```diff
@@ -98,6 +98,7 @@ def __init__(
         version: Optional[Version] = None,
         credentials: Dict[str, Any] = None,
         fs_args: Dict[str, Any] = None,
+        metadata: Dict[str, Any] = None,
     ) -> None:
         _fs_args = deepcopy(fs_args) or {}
         _fs_open_args_load = _fs_args.pop("open_args_load", {})
```

Comment:
I just noticed this dataset is missing some docstring to explain the arguments. Might be good to add that now it's being updated.